src/viur/core/secret.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from google.cloud import secretmanager
+from viur.core.config import conf
+
+"""
+This module provides utiliy functions for accessing values stored in the Google Cloud Secret Manager.
+"""
+
+# Global secret manager client instance
+__client = secretmanager.SecretManagerServiceClient()
+
+
+def get(secret: str, version: str = "latest") -> str:
```

Comment:
Since the versions are incremented `int`s and we are using string interpolation, this should be fine too.


```suggestion
def get(secret: str, version: int | str = "latest") -> str:
```